assets/js/modules/analytics/components/dashboard/DashboardAllTrafficWidget/UserCountGraph.js

```diff
@@ -100,12 +107,15 @@ UserCountGraph.chartOptions = {
 		startup: true,
 	},
 	curveType: 'function',
-	height: 340,
+	height: 410,
```

Comment:
There's now a lot of extra whitespace at the bottom of the widget above the "Source" link, which looks odd. Likely due to increasing the height here? Can we achieve the vertical alignment without changing the overall chart height - why was this necessary?

See screenshot of extra whitespace:
![Screen Shot 2021-01-28 at 3 36 11 PM](https://user-images.githubusercontent.com/3531426/106212123-c2636700-617e-11eb-98b7-1d51a375f00b.png)

Reply:
This is as tight as I can get the margins without changing the grid/widget margins on things:

<img width="1138" alt="Screenshot 2021-01-29 at 13 32 24" src="https://user-images.githubusercontent.com/90871/106282052-f8185680-6237-11eb-9e61-4b258ba10110.png">

The margins are accounting for most of the whitespace:

<img width="1157" alt="Screenshot 2021-01-29 at 13 45 02" src="https://user-images.githubusercontent.com/90871/106282230-357ce400-6238-11eb-9e4e-87153f37bda7.png">
<img width="1190" alt="Screenshot 2021-01-29 at 13 44 53" src="https://user-images.githubusercontent.com/90871/106282234-36ae1100-6238-11eb-8294-699b541fc55b.png">

But I think my reductions in the margins is an improvement 🙂